activate.sh

```diff
@@ -295,7 +295,7 @@ autoenv_leave() {
 	_files=$(
 		command -v chdir >/dev/null 2>&1 && chdir "${from_dir}" || builtin cd "${from_dir}"
 		_hadone=''
-		while [ "$PWD" != "" ] && [[ $to_dir != $PWD* ]]; do
+		while [ "$PWD" != "" ] && [ "$PWD" != "/" ] && [[ $to_dir/ != $PWD/* ]]; do
```

Comment:
we need to special case `/` because `$to_dir/ != $PWD/*` does not handle `/` correctly (e.g., if `$to_dir` is `/a` and `$PWD` is `/`, the check says that `/a/` is not a prefix of `//`, but in fact `/a/` is a child of `//`)

Reply:
Makes sense, thanks for the explanation